eduardo/access-modifiers/training.rb

```diff
@@ -0,0 +1,68 @@
+class Training
+#class variable
+  @@training_name = 'Full stack course'
+  @@company = 'TangoSource '
+#class method
+  def self.course_name
+    @@training_name
+  end
+
+  def self.current_time
+    print "#{Time.now}"
+  end
+
+  def self.course_information
+    print "The #{course_name} is imparted for #{@@company}"
+    puts
+    print 'The course will take 3 months length and is free'
+  end
+
+
+  attr_reader :teachers, :name, :students, :grades
+
+  def initialize(name, student_capacity)
+    @name = name
+    @student_capacity = student_capacity
+    @teachers = []
+    @students = []
+    @grades = Hash.new { |h, k| h[k] = [] }
+  end
+
+  def hire_teacher(teacher_name)
+    @teachers << teacher_name
+  end
+
+  def enroll_course(student_name)
+    @student_capacity > @students.length ? @students << student_name : false
+    print "#{student_name} was added to course successfuly, Current student capacity is #{@student_capacity -= 1}"
```

Comment:
Also, I think you can `print` or `puts` first, then, do the logic, so the returned value would be the result of the operation.